vispy/visuals/markers.py

```diff
@@ -474,7 +474,6 @@
     '^': triangle_up,
     'v': triangle_down,
     '*': star,
-    None: None,
 }
 marker_types = tuple(sorted(list(_marker_dict.keys())))
 
```

Comment:
Add `_marker_dict[None] = None` here and use `_check_valid('symbol', symbol, _marker_dict)` in `set_symbol()`